### PR TITLE
CDAP-7182 Document using cdap script

### PR DESCRIPTION
**Pull request by wolf31o2**
Updated docs to use the new unified `cdap` scripts.

```
sed -i \
    -e 's#cdap-cli.sh#cdap cli#g' \
    -e 's#cdap.sh#cdap sdk#g' \
    -e 's#cdap.bat#cdap.bat sdk#g' \
    -e 's#cdap-cli.bat#cdap.bat cli#g' \
    */*.rst
```

This assumes that caskdata/cdap#6774 has been merged.

### CODE REVIEW

File: MovieRecommender/README.rst [hunk 1]
```diff
@@ -44,34 +44,34 @@ From the project root, build ``MovieRecommender`` with `Apache Maven <http://mav
 
   $ MAVEN_OPTS="-Xmx512m" mvn clean package
 
-Note that the remaining commands assume that the ``cdap-cli.sh`` script is available on your PATH.
+Note that the remaining commands assume that the ``cdap cli`` script is available on your PATH.
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: I suggest saying instead "`cdap` script" as that is the name of the script.

```
Note that the remaining commands assume that the ``cdap`` script is available on your PATH.
```

File: Netlens/README.rst [hunk 2]
```diff
@@ -79,37 +79,37 @@ From the project root, build ``Netlens`` with `Apache Maven <http://maven.apache
 
   $ MAVEN_OPTS="-Xmx512m" mvn clean package
 
-Note that the remaining commands assume that the ``cdap-cli.sh`` script is available on your PATH.
+Note that the remaining commands assume that the ``cdap cli`` script is available on your PATH.
```
Comment (awholegunch) [inline Netlens/README.rst]: See above (common across all files).

File: MovieRecommender/README.rst (hunk 1 above)
Comment (wolf31o2) [reply MovieRecommender/README.rst]: I agree.

File: Netlens/README.rst (hunk 2 above)
Comment (wolf31o2) [reply Netlens/README.rst]: Done.

**Comment by awholegunch** [comment]
One other (quick change): update the copyrights to "-2016" in these four files, if they aren't already. Aside from that, LGTM.

**Pull request merged**